config/defaults.mk

```diff
@@ -189,15 +189,18 @@ OPENMP_LIB =
 POSIX_CLOCKS_LIB = -lrt
 
 # SUNDIALS library configuration
-# For sundials_nvecparhyp and nvecparallel remember to build with MPI_ENABLED=ON
+# For sundials_nvecmpiplusx and nvecparallel remember to build with MPI_ENABLE=ON
 # and modify cmake variables for hypre for sundials
 SUNDIALS_DIR = @MFEM_DIR@/../sundials-5.0.0/instdir
 SUNDIALS_OPT = -I$(SUNDIALS_DIR)/include
-SUNDIALS_LIB = -Wl,-rpath,$(SUNDIALS_DIR)/lib64 -L$(SUNDIALS_DIR)/lib64\
+SUNDIALS_LIB = $(XLINKER)-rpath,$(SUNDIALS_DIR)/lib -L$(SUNDIALS_DIR)/lib\
```

Comment:
You would have to tell sundials to put the libraries in `lib` with the cmake option `CMAKE_INSTALL_LIBDIR=lib`, because we still default to wherever GNUInstallDirs decides. However, I will change this to use the `wildcard` command like @v-dobrev suggested.

Reply:
Fixed.